website/src/components/blog/BlogPost.tsx

```diff
@@ -46,9 +46,17 @@ export const BlogPost: React.FunctionComponent<Props> = ({
                         post.frontmatter.title
                     )}
                 </h1>
-                <p className="blog-post__byline mb-0">
-                    {post.frontmatter.author} on {post.frontmatter.publishDate}
-                </p>
+                {post.frontmatter.author && post.frontmatter.publishDate && (
+                    <p className="blog-post__byline mb-0">
+                        {post.frontmatter.authorUrl ?
+                            <a href={post.frontmatter.authorUrl} target="_blank" rel="nofollow noreferrer">
+                                {post.frontmatter.author}
+                            </a> :
```

Comment:
In the future, this may require an internal `<Link>` if employee profiles end up within the scope of About. However, we don't have a case for this at this time, because all of the links are outgoing.

Reply:
Yes, maybe evaluating when there's an outside link with `http` in the url and using a `Link` or `a` tag accordingly